contents/blog/the-posthog-array-1-33-0.md

```diff
@@ -0,0 +1,124 @@
+
+### New: Experimentation
+
+A full end-to-end A/B testing suite is now part of PostHog! Feature Flags were already powerful and you could technically use them to run A/B tests, but with a lot of manual work. This new feature will help you plan your experiment (select users, determine sample size exposure, determine estimated run time), allocate users evenly across test groups, track results seamlessly and let you know once your results reach statistical significance.
+
+<img src="https://posthog-static-files.s3.us-east-2.amazonaws.com/Website-Assets/Array/1_33_0-experiments.png" alt="Screenshot: Experiments view" />
+
+With Experimentation, you can now test multiple variants to optimize for a specific metric, or even conversion rate for a funnel. You'll now be able to ship product changes confidently while maintaining speed.
+
+<img src="https://posthog-static-files.s3.us-east-2.amazonaws.com/Website-Assets/Array/1_33_0-experiments-2.png" alt="Screenshot: Experiments with a funnel conversion rate target metric" />
+
+> 🎁 Experimentation is a premium feature and requires a PostHog Scale or Enterprise license. [Learn more](/pricing).
+
+<br />
+
+
+### New: Dashboard permissions
+
+To introduce more detailed control to dashboards for large teams, PostHog now lets you select different edit permissions for each dashboard. You can select whether anyone can edit a dashboard or just a specific list of team members. This can help prevent accidental edits and provide confidence that the dashboards are showing the metrics you intend. Underlying insights in a dashboard are also protected from edit when this setting is enabled.
+
+
+<img src="https://posthog-static-files.s3.us-east-2.amazonaws.com/Website-Assets/Array/1_33_0-dashboard-permissions.png" alt="Screenshot: Dashboard permissions" />
+
+> 🎁 Dashboard permissions is a premium feature and requires a **PostHog Enterprise** license. [Learn more](/pricing).
+
+<br />
+
+### New: Altitiny Cloud support
+
+We've partnered up with Altinity to offer a worry-free managed ClickHouse database. Altinity are experts in Clickhouse and data infrastructure and they help large organizations to deploy, manage and scale their PostHog instance.
+
+Find out more about our Altinity integration in our [Marketplace docs](/marketplace/altinity).
+
+<br />
+
+### New: Instance configuration UI
+
+We're significantly improving the way you manage your PostHog instance. Some settings are now configurable directly on the UI, instead of you having to rely on environment variables (e.g. email settings). You can read more about these settings in the [instance settings docs](/docs/self-host/configure/instance-settings) and [environment variables docs](/docs/self-host/configure/environment-variables).
+
+<img src="https://posthog-static-files.s3.us-east-2.amazonaws.com/Website-Assets/Array/1_33_0-instance-configuration.png" alt="Screenshot: Instance configuration" />
+
+<br />
+
+
+### Other improvements & fixes
```

Comment:
Missing: Async migrations (need to run async migration to speed things up before next release). cc @yakkomajuri

Reply:
Resolving - the other thread paolo set up actually covers this